README.md

```diff
@@ -84,6 +89,27 @@ Script file to set environment variables in Linux/Mac
 cd .../go_ibm_db/installer
 source setenv.sh
 
+For Docker Linux Container, use below commands
+yum install -y gcc python python-devel git go wget pam tar xz php php-devel make gcc-c++ php-pear python3 python3-devel
+cd /root
+wget -c https://bootstrap.pypa.io/get-pip.py
```

Comment:
This block need to be changed. pip and python is not required for go driver installation.